model-engine/model_engine_server/entrypoints/populate_llm_fine_tuning_job_repository.py

```diff
@@ -0,0 +1,472 @@
+            "gpus": 2 if cloud_provider == "azure" else 4,
+            "gpu_type": "nvidia-ampere-a10",
+        },
+        "llama_2_34b": {
+            "cpus": 60,
+            "memory": "400Gi",
+            "storage": "300Gi",
+            "gpus": 4,
+            "gpu_type": "nvidia-ampere-a100e",
+        },
+        "llama_2_70b": {
+            "cpus": 80,
+            "memory": "1000Gi",
+            "storage": "500Gi",
+            "gpus": 8,
+            "gpu_type": "nvidia-ampere-a100e",
+        },
+    }
+
+    name = BUNDLE_NAME_BY_MODEL[model_type]
+    resource_requests = RESOURCE_REQUESTS_BY_MODEL[model_type]
+
+    response = requests.post(
+        f"{url}/v1/docker-image-batch-job-bundles",
+        json={
+            "name": name,
+            "image_repository": "spellbook-finetune",
+            "image_tag": image_tag,
+            "command": [
+                "dumb-init",
+                "--",
+                "ddtrace-run",
+                "python",
+                "llm/finetune_pipeline/docker_image_fine_tuning_entrypoint.py",
+                "--config-file",
+                "/launch_reserved/config_file.json",
+            ],
+            "mount_location": "/launch_reserved/config_file.json",
+            "resource_requests": resource_requests,
+            "public": True,
+        },
+        headers={"Content-Type": "application/json"},
+        # Use Spellbook api key
+        auth=requests.auth.HTTPBasicAuth("63bf1777e487096169d50214", ""),
+    ).json()
+    return response["docker_image_batch_job_bundle_id"]
+
+
+async def main(args):
+    env = args.env
+    cloud_provider = args.cloud_provider
+    url = args.url or f"http://model-engine.{hmi_config.gateway_namespace}.svc.cluster.local"
+    repository = args.repository or hmi_config.cloud_file_llm_fine_tune_repository
+    initialize_repository = args.initialize_repository
+
+    if repository.startswith("s3://"):
+        repo = S3FileLLMFineTuneRepository(file_path=repository)
+    else:
+        repo = ABSFileLLMFineTuneRepository(file_path=repository)
+
+    # Clears the file. Needed the first time we're populating data
+    if initialize_repository:
+        await repo.initialize_data()
+
+    lora_7b_or_13b_bun = create_model_bundle(cloud_provider, url, "7b_or_13b", FT_IMAGE_TAG)
+    print(f"lora_7b_or_13b bundle id: {lora_7b_or_13b_bun}")
+
+    lora_llama_2_34b_bun = create_model_bundle(cloud_provider, url, "llama_2_34b", FT_IMAGE_TAG)
+    print(f"lora_34b_bun bundle id: {lora_llama_2_34b_bun}")
+
+    lora_llama_2_70b_bun = create_model_bundle(cloud_provider, url, "llama_2_70b", FT_IMAGE_TAG)
+    print(f"llama_2_70b bundle id: {lora_llama_2_70b_bun}")
+
+    if env == "training":
+        await repo.write_job_template_for_model(
+            "test_base_model",
+            "test_fine_tuning_method",
+            LLMFineTuneTemplate(
+                docker_image_batch_job_bundle_id="batbun_cioa1g58sre002btvid0",
```

Comment:
feel like this hardcoded value doesn't work, should we just remove the test bundle here?